src/lib/processors/watch-processor.ts

```diff
@@ -89,7 +89,7 @@ const eventToPhaseMap = {
 export function setupWatch(capabilities: Capability[], ignoredNamespaces?: string[]): void {
   for (const capability of capabilities) {
     for (const binding of capability.bindings.filter(b => b.isWatch)) {
-      runBinding(binding, capability.namespaces, ignoredNamespaces);
+      void runBinding(binding, capability.namespaces, ignoredNamespaces);
```

Comment:
`runBinding` is `async` and it looks like this function may not be treating it as an async function? This may be an implementation detail for the maintainers to handle. If it gets too complex to look at, toss an issue into the backlog.